homeassistant/components/number/const.py

```diff
@@ -103,6 +104,12 @@ class NumberDeviceClass(StrEnum):
     Unit of measurement: `UnitOfPressure` units
     """
 
+    AREA = "area"
```

Comment:
Still not alphabetical

Reply:
You marked this as resolved - but it is still wrong.
Did you forget to push an updated?

Reply:
Possibly, let me check

Reply:
Yeah forgot to push a commit